packages/wrangler/src/inspect.ts

```diff
@@ -539,5 +575,7 @@ function logConsoleMessage(evt: Protocol.Runtime.ConsoleAPICalledEvent): void {
     }
   }
 
-  console[evt.type](...args);
+  const method = mapConsoleAPIMessageTypeToConsoleMethod[evt.type];
+  //@ts-expect-error I dunno how to make this type pass lol
```

Comment:
🤪 halp

Reply:
You need to exclude the constructor from the list of keys:

```ts
export const mapConsoleAPIMessageTypeToConsoleMethod: {
  [key in Protocol.Runtime.ConsoleAPICalledEvent["type"]]: Exclude<
    keyof Console,
    "Console"
  >;
```

But then TS complains that `args` must be a tuple or be passed to a rest argument.
We can work around that by using `apply()` (telling eslint not to stress):

```ts
  // eslint-disable-next-line prefer-spread
  console[method].apply(console, args);
```